src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -107,38 +109,47 @@ private static async Task InitializeInstance(ITestOutputHelper output)
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
                 psi.FileName = "cmd";
-                psi.Arguments = $"/c npm {psi.Arguments}";
-            }
-
-            // It's important that we get the folder value before we start the process to prevent
-            // untracked processes when the tracking folder is not correctly configure.
-            var trackingFolder = GetProcessTrackingFolder();
-            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("helix")))
-            {
-                // Just create a random tracking folder on helix
-                trackingFolder = Path.Combine(Directory.GetCurrentDirectory(), Path.GetRandomFileName());
-                Directory.CreateDirectory(trackingFolder);
-            }
-            
-            if (!Directory.Exists(trackingFolder))
-            {
-                throw new InvalidOperationException($"Invalid tracking folder. Set the 'SeleniumProcessTrackingFolder' MSBuild property to a valid folder.");
+                var cmd = (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("helix"))) ? "npm" : "yarn";
+                psi.Arguments = $"/c {cmd} {psi.Arguments}";
             }
 
             Process process = null;
             Process sentinel = null;
             string pidFilePath = null;
-            try
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("helix")))
```

Comment:
Super nit: We should probably have a utility `IsHelix` function?

Reply:
Yeah there is one I think but it lives in the shared testing sources, while it doesn't bring in any dependencies to just check the environment, so that's the trade off